plugins/source.go

```diff
@@ -103,7 +103,7 @@ func NewSourcePlugin(name string, version string, tables []*schema.Table, newExe
 }
 
 func (p *SourcePlugin) SetLogger(logger zerolog.Logger) {
-	p.logger = logger
+	p.logger = logger.With().Str("module", p.name+"-src").Logger()
```

Comment:
shouldn't it be on the serve side? i.e set outside? but this is also fine I guess.